deepspeed/utils/nvtx.py

```diff
@@ -5,15 +5,19 @@
 
 from deepspeed.accelerator import get_accelerator
 
+enable_nvtx = True
```

Comment:
if we set it based on the config as I mentioned above, you will not have to set it to True here